README.md

```diff
@@ -1216,7 +1217,10 @@ VSC8552; this is indicated with `--` in the relevant table positions.
 
 ### `humility openocd`
 
-This command launches OpenOCD based on the config file in a build archive
+This command launches OpenOCD based on the config file in a build archive,
+which then allows one to connect with either GDB or directly via telnet.
+If the intention is to only run GDB, note that `humility gdb --run-openocd`
+will both run OpenOCD and run a foreground GDB that is connected to it.
```

Comment:
Fun fact: this change isn't due to me at all, but when I ran `cargo xtask readme`, it was generated. Guess that hadn't been done in a while.